fpdf/html.py

```diff
@@ -456,8 +456,8 @@ def handle_starttag(self, tag, attrs):
             self.heading_level = int(tag[1:])
             hsize = self.heading_sizes[tag]
             self.pdf.set_text_color(150, 0, 0)
+            self.pdf.ln(self.h + 0.2 * hsize)  # 20 % more space above heading
```

Comment:
Could you move this constant in a class property please?

Reply:
ok